### DISCUSSION TITLE
Is it possible to hide PointerLock message?

### DISCUSSION THREAD

**Discussion by basdp** [MicrosoftEdge/WebView2Feedback#4050] [Q&A]
I have an app that I want to use pointerlock with (to hide the mouse while 'turning' a knob, like in audio applications). the PointerLock API works perfectly, although it shows a banner overlay to 'Press ESC to show your cursor'. Is it possible to hide that overlay in WebView2?

**Comment by champnic** [comment]
Hey @basdp - There currently isn't a way to do this. Do you want me to change this to a feature request to track on our backlog?

**Comment by basdp** [comment]
Hey @champnic, thanks for reaching out. Would be cool if we can change this to a feature request! I don't know if this is a feature a lot of people are waiting for, but the pointer lock message is at least in Electron explicitly hidden. For games and entertainment type apps, this is a very welcome feature!

**Comment by champnic** [comment]
Done - added as a feature request.

Do you know if Electron has this behavior by default? Or is there an API for that?

**Comment by stefan1000** [comment]
firefox at least has a timeout that can be customized (pointer-lock-api.warning.timeout).

(someone asking how to implement it on electron, I haven't verified but I guess electron won't show those messages).
https://stackoverflow.com/questions/58731443/how-to-enalbe-press-esc-to-exit-full-screen-mode-in-electron

This is an important feature for us since this "Press ESC" message overlays important UI controls in our application and is not really needed for a desktop application. A setting to disable it would be nice, we don't need an API to control the dialog (if we need it we will implement it using our own UI).

**Comment by John-Dean** [comment]
This is still something that is requested!

For reference this is not something Electron has in any capacity, and it's a blocker to making things like games using WebView.

**Comment by castorix** [comment]
On Windows, it can be hidden with a _WH_CBT_ hook and intercepting _HCBT_CREATEWND_, but it is complicated.
(tested in WinUI3 / C#)
